package.json

```diff
@@ -5,12 +5,14 @@
   "main": "index.js",
   "scripts": {
     "start": "babel-node src/index.js",
-    "test": "npm run setup:test && NODE_ENV=test nyc --reporter=html --reporter=text --reporter=lcov ./node_modules/.bin/mocha  ./tests/* --require @babel/register --timeout 4000 --exit",
+    "test": "NODE_ENV=test nyc --reporter=html --reporter=text --reporter=lcov ./node_modules/.bin/mocha  ./tests/* --require @babel/register --timeout 4000 --exit",
     "dev": "NODE_ENV=development nodemon --exec babel-node src/index.js",
```

Comment:
what did you change the test script??

Reply:
This is the one for develop I updated .The changes are showing the previous script from my branch. Check develop you will realize it is the same script